### PR TITLE
Restore invokeai-update

### PR DESCRIPTION
**Pull request by lstein**
At some point `pyproject.toml` was modified to remove the invokeai-update and invokeai-model-install scripts. This PR fixes the issue.

If this was an intentional change, let me know and we'll discuss.

### CODE REVIEW

**Comment by lstein** [comment]
I'm also taking the opportunity to bump the version number to 2.3.2+a0. I think we've reached the critical mass to do a bug fix release.

File: ldm/invoke/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__='2.3.1.post2'
+__version__='2.3.2.dev0'
```
Comment (mauwii) [inline ldm/invoke/_version.py:1]: are you sure .dev0 will work with pep - shouldn't it be `2.3.2-a0` 🤔

**Review by mauwii — APPROVED** [review]

File: ldm/invoke/_version.py (hunk 1 above)
Comment (lstein) [reply ldm/invoke/_version.py:1]: I dunno. All these PEPs confuse me. Here's what PEP-440 says:

```
Public version identifiers are separated into up to five segments:

Epoch segment: N!
Release segment: N(.N)*
Pre-release segment: {a|b|rc}N
Post-release segment: .postN
Development release segment: .devN
```

Comment (lstein) [reply ldm/invoke/_version.py:1]: In any case, I'm bumping to 2.3.2.

**Comment by lstein** [comment]
I'm overriding the codeowner review because the change is safe and I need to get the release out. I'll take responsibility for breaking anything. :-)

**Review by mauwii — APPROVED** [review]

**Pull request merged**